components/brave_wallet/browser/eth_data_parser.cc

```diff
@@ -15,6 +15,31 @@ namespace brave_wallet {
 
 namespace {
 
+namespace erc1155 {
+
+// GetBytesHexFromSafeTransferFromData is only applicable when the bytes are the
```

Comment:
nit: For ERC1155::SafeTransferFrom, `bytes` is the fifth and final parameter and the preceding 4 params are fixed types exactly 32 bytes.

Reply:
Updated function name and comment.